starcli/search.py

```diff
@@ -27,26 +29,44 @@
     "valid": "The request returned successfully, but an unknown exception occurred.",
 }
 
+FORMAT = "%(message)s"
+
+
+
+httpclient_logger = logging.getLogger("http.client")
+
+def httpclient_logging_debug(level=logging.DEBUG, debug_level=1):
```

Comment:
I don't think the `debug_level` parameter used here? The level is set to 1 on line 44

Reply:
I am removing the debug_level parameter. I need to keep the "level" parameter as it is used.